docker-compose.db.yml

```
@@ -9,14 +9,24 @@ services:
       - POSTGRES_PASSWORD=${DB_PASSWORD}
     ports:
-      - "${DB_PORT}:${DB_PORT}"
+      - "${DB_PORT}:5432"
     volumes:
       - postgres-data:/var/lib/postgresql/data
     networks:
       - my-network
 
+  redis:
+    image: redis:7.2
+    command: [ "redis-server" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)